css/layout.css

```diff
@@ -0,0 +1,50 @@
+.hidden {
+  display: none !important;
+}
+
+.container {
+  max-width: 575px;
+  height: 100vh;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+}
+
+/* .row {
+  display: flex;
+  flex-wrap: wrap;
+} */
+
+.row-full {
+  display: flex;
+  flex-direction: column;
+  flex-basis: 100%;
+}
+
+.row-half {
+  display: flex;
+  flex-direction: column;
+  flex-basis: 50%;
+}
+
+.row-three-fourths {
+  display: flex;
+  flex-direction: column;
+  flex-basis: 75%;
+}
+
+.row-one-fourth {
+  display: flex;
+  flex-direction: column;
+  flex-basis: 25%;
+}
```

Comment:
Why are there so many types of rows? Shouldn't this sizing be handled by `column` class variants?